### ISSUE TITLE
Document avoiding warning from private-call linter

### ISSUE THREAD

**Issue by borkdude** [clj-kondo/clj-kondo#837]
To disable this linter in specific namespaces you can use one of these two ways:

```
(ns foo)
(defn- foo [])
(foo)
(ns bar
  {:clj-kondo/config {:linters {:private-call {:level :off}}}} ;; 1
  (:require [foo]))
(foo/foo)   ;; 1
(#'foo/foo) ;; 2
```

**Comment by lvh** [comment]
The issue description suggests that `(#'foo/foo) ;; 2` is a way to avoid the problem, but I definitely seen the linter fire by calling the var. Am I misinterpreting the description?

**Comment by borkdude** [comment]
@lvh Repro please? This should not be the case.

**Comment by lvh** [comment]
Sorry: I'm totally wrong. I ended up putting the ns-specific meta in the wrong ns. My bad!

**Comment by lvh** [comment]
Can I specifically exclude a namespace from this linter in my clj-kondo config, the way you can with `:unresolved-symbol`?

**Comment by borkdude** [comment]
@lvh Do you mean disable this linter within a specific namespace? Or disable vars from a specific namespace?

**Comment by lvh** [comment]
Well, either one really. FWIW it's possible this issue will go away with a future macro-eval change, but: I have a macro that included a ``some-namespace/a-private-var` inside a varquote (https://github.com/lvh/fix-default-branch/blob/trunk/test/io/lvh/fix_default_branch/test_harness.clj#L25). clj-kondo complained that was a private call, even though it's really just a symbol (that happens to point to a private var).

**Comment by borkdude** [comment]
@lvh Thanks, this is a bug.

Repro:

```
(ns foo)

(defn- foo [])


(ns bar
  (:require [foo]))

(prn `foo/foo)
```

This is fine Clojure, clj-kondo should not complain about it. I'll make a separate issue for it.

**Comment by borkdude** [comment]
@lvh This issue should be fixed in this new release: https://github.com/borkdude/clj-kondo/releases/tag/v2020.06.21
